i18n/translations/en.json

```diff
@@ -391,7 +392,6 @@
   "sort": "Sort",
   "title": "Title",
   "template": "Template",
-  "today": "Today",
```

Comment:
Have you checked if we use that string anywhere else? It could potentially become useful in the future as well, so I'm not sure if we should remove it.

Reply:
We don't use it elsewhere. But you are right, let's not throw away the translations.